pages/common/complete.md

```diff
@@ -0,0 +1,15 @@
+# complete
+
+> Provides argument autocompletion to shell commands.
+
+- Apply an autocomplete function to a command:
+
+`complete -F {{function}} {{command}}`
+
+- Apply an autocomplete command to a command:
+
+`complete -C {{autocomplete_command}} {{command}}`
+
+- Apply autocompletion without appending a space to the completed word:
+
+`complete -o nospace -F {{function}} {{command}}`
```

Comment:
Wasn't sure how many examples I should include. I've only used `complete` at the basic level, so I'd love to hear any thoughts on any additional examples we should or shouldn't include here.

Reply:
This is fine.

Reply:
Up to 8 is fine!